### DISCUSSION TITLE
exa vs eza look

### DISCUSSION THREAD

**Discussion by jonas-frey** [eza-community/eza#194] [Q&A, Howto, Help] [answered]
I just installed eza (v0.11.0 via ubuntu repos mentioned on homepage) and noticed some differences to exa that I'm not sure whether they're on purpose. Consider the following screenshots.

![Screenshot_20230904_222948](https://github.com/eza-community/eza/assets/31709069/58116acd-c8be-4131-afe4-7303bf54af04)

![Screenshot_20230904_223031](https://github.com/eza-community/eza/assets/31709069/43d72eaa-7d33-435b-ae7b-47d0d60094ab)

Besides the changed color, note the different icons for `tex` files, `main.out`, and the Makefile. Is that deliberate? Why the arrow symbol for tex files instead of the T?

**Comment by OmiChrome** [comment]
I think EXA color looks better because of 
Violet 💜✨

**Reply by cafkafk** [reply]
The colors being different is likey due to https://github.com/eza-community/eza/pull/64. If you hate the colors, you should be able to change them, check out `man eza_colors`.

**Reply by jonas-frey** [reply]
Is there a quick way to get the old color scheme back? I'm too lazy to set rgb values one by one.

**Reply by ariasuni** [reply]
There’s no easy way… tho you can change the colors of your terminal, since eza follows these now.

**Reply by cafkafk** [reply]
also if your problem is punctuation color, we will probably change that since a lot of people have said it broke their setup https://github.com/orgs/eza-community/discussions/209

**Comment by gierens** [comment]
Hi @jonas-frey,

so exa and by extension also eza use symbols from the unicode private Use domain for their icons. exa uses this font-size symbol and when I skimmed the private use icons I found a dedicated TeX symbol and thus replaved that. This is how it looks on my system for example:
![Screenshot from 2023-09-05 11-53-42](https://github.com/eza-community/eza/assets/49617392/f61a84b1-4f44-4349-bf96-420022e806d3)

I think the issue here is, that fonts are allowed to use this private unicode space in any way they like. There is no real convention what the different characters should to represent, and so the font you are using apparently does something else with the character. May I ask what font you are using?

@cafkafk does the TeX icon look fine on your system?

Also, as a reference I just created a gist containing a list of all private use code points so anyone can check what symbols their font displays and how: https://gist.github.com/gierens/3c7ce960eea7feb1ac53fa110f950ef6

As a side note, we also just created a dedicated discussion regarding icons: https://github.com/orgs/eza-community/discussions/196

**Reply by cafkafk** [reply]
> @cafkafk does the TeX icon look fine on your system?

Yeop!

![2023-09-05_12-36](https://github.com/eza-community/eza/assets/89321978/d2a20949-0f64-476e-9945-bbc5caac9b97)

**Reply by daluar657** [reply]
(Hi @cafkafk ! What font do you use? Looks gorgeous)

**Reply by cafkafk** [reply]
> (Hi @cafkafk ! What font do you use? Looks gorgeous)

([Fantasque Sans Mono](https://github.com/belluzj/fantasque-sans), and yes it is lovely)

**Comment by cafkafk** [comment, marked as answer]
> Is that deliberate?

No, that would be super ugly. It might be the case that you're on Nerdfonts 2 and not Nerdfonts 3? 

See: https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.0.0

**Reply by jonas-frey** [reply]
Ahh yes it was  an old version of nerd fonts. Thanks!
